src/building/building.c

```diff
@@ -14,6 +14,7 @@
 #include "core/array.h"
 #include "core/log.h"
 #include "figure/formation_legion.h"
+#include "industry.h"
```

Comment:
Please add the path to it (building/industry.h) and place it in the alphabetical order of the includes.